.eslintrc

```diff
@@ -2,8 +2,10 @@
   "env": {
     "node": true
   },
-  "extends": "airbnb",
+  "extends": "airbnb-base",
```

Comment:
The updated airbnb package was requiring jsx and react plugins.  Pointed us instead to the base package.